arch/arm/soc/st_stm32/stm32l4/dts.fixup

```diff
@@ -74,12 +74,22 @@
 #define FLASH_DEV_BASE_ADDRESS		        ST_STM32L4_FLASH_CONTROLLER_40022000_BASE_ADDRESS
 #define FLASH_DEV_NAME			        ST_STM32L4_FLASH_CONTROLLER_40022000_LABEL
 
+#if defined(CONFIG_SOC_STM32L432XC)
+#define CONFIG_USB_BASE_ADDRESS			ST_STM32_USB_40006800_BASE_ADDRESS
+#define CONFIG_USB_IRQ				ST_STM32_USB_40006800_IRQ_USB
+#define CONFIG_USB_IRQ_PRI			ST_STM32_USB_40006800_IRQ_USB_PRIORITY
+#define CONFIG_USB_NUM_BIDIR_ENDPOINTS		ST_STM32_USB_40006800_NUM_BIDIR_ENDPOINTS
```

Comment:
Please add here:
```
#define CONFIG_USB_NUM_IN_ENDPOINTS		ST_STM32_USB_40006800_NUM_IN_ENDPOINTS
#define CONFIG_USB_NUM_OUT_ENDPOINTS		ST_STM32_USB_40006800_NUM_OUT_ENDPOINTS
```
Someone may override the DT usb node on their project and choose some endpoints to be only OUT/IN.